com.mattermost.Desktop.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id": "com.mattermost.Desktop",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "18.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "mattermost-flatpak",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--filesystem=xdg-run/dconf",
```

Comment:
Also, are you sure you need all these `dconf`-related?

Additionally, you might not need `"--filesystem=home"` – guess you should be able to get around with `"--filesystem=xdg-download"` only. My understanding is that Flatpaks have read-only access to the user's home directory, so that comes handy for uploading stuff, and the `"--filesystem=xdg-download"` to finally close the loop and be able to download/persist anything that's shared with you.

Reply:
I searched my dconf setttings and there is no entry for Mattermost, so it is not neccessary I think. Thanks for pointing that out. However there is config in `~/.config/Mattermost` so I believe that I need write permission to home filesystem. Am I right?

Reply:
That one should go/map automatically to `$HOME/.var/.config`, if that folk honors the XDG spec. Give it a try with `"--filesystem=xdg-download"` only, it should work.

Reply:
With `"--filesystem=xdg-download"` only I get correct behaviour in terms of settings, but I can't upload files from other location than `~/Downloads`. With `"--filesystem=home"` I can upload all files from my home.

Reply:
@TingPing may I kindly ask your opinion on this? :slightly_smiling_face:

Reply:
I'd prefer to have it limited to xdg-downloads, users can always override it themselves.

Reply:
I tried and I don't agree we should do it that way. Downloading file dialog allows you to choose any directory in your home and confirming it with the `Save` button doesn't show any error - but the file is then just not there. In my opinion that is very confusing behaviour for users. Does anyone have a better idea than to keep `--filesystem="home"` switch?